social/app/models/post.py

```diff
@@ -45,6 +45,7 @@ class Post(models.Model):
     source = models.URLField()
     origin = models.URLField()
     description = models.TextField()
+    github_id = models.TextField(default="")
```

Comment:
ID isn't a URL, though. It's something like "tag:github.com,2008:PushEvent/5643591992"